README.md

```diff
@@ -20,20 +20,62 @@ Language mappings are defined in `.tx/config`, with the second code being the on
 
 # Building
 
-### Dependencies
-- Android SDK, with `$ANDROID_HOME` pointing to it (you can skip this if you are using Android Studio)
-- Android NDK (you should install the required version (`ext.ndkVersionShared` in `./build.gradle`) with the usual tools, such that it's located within `$ANDROID_HOME/ndk/`).
-- Go (see [here](https://docs.syncthing.net/dev/building.html#prerequisites) for the required version)
-- Java Version 11 (you scan skip this if you are using Android Studio, otherwise you might need to set `$JAVA_HOME` accordingly)
+## Dependencies
 
-### Build instructions
+These are necessary for building from the command line. If you build using
+Docker, you don't need to set them up separately.
 
-Make sure you clone the project with
-`git clone https://github.com/syncthing/syncthing-android.git --recursive`. Alternatively, run
-`git submodule init && git submodule update` in the project folder.
+1. Android SDK and NDK
+    1. Download SDK command line tools from https://developer.android.com/studio#command-line-tools-only.
+    2. Unpack the downloaded archive to an empty folder. This path is going
+       to become your `ANDROID_HOME` folder.
+    3. Inside the unpacked `cmdline-tools` folder, create yet another folder
+       called `latest`, then move everything else inside it, so that the final
+       folder hierarchy looks as follows.
+       ```
+       cmdline-tools/latest/bin
+       cmdline-tools/latest/lib
+       cmdline-tools/latest/source.properties
+       cmdline-tools/latest/NOTICE.txt
+       ```
+    4. Navigate inside `cmdline-tools/latest/bin`, then execute
+       ```
+       ./sdkmanager "platform-tools" "build-tools;31.0.0" "platforms;android-31" "extras;android;m2repository" "ndk;25.1.8937393"
+       ```
+       The required tools and NDK will be downloaded automatically.
 
-Build Syncthing using `./gradlew buildNative`. Then use `./gradlew assembleDebug` or
-Android Studio to build the apk.
+        **NOTE:** You can double-check [Dockerfile](docker/Dockerfile) to make
+        sure that the version numbers are up-to-date.
+2. Go (see https://docs.syncthing.net/dev/building#prerequisites for the
+   required version)
+3. Java version 11
+4. Python version 3
+
+## Build instructions
+
+### Command line
```

Comment:
There's no other build instructions, so could just drop this sub-header.

Reply:
Done.